build.sbt

```diff
@@ -48,6 +48,7 @@ TwirlKeys.templateImports ++= Seq(
   "lila.common.String.html._"
 )
 resourceDirectory in Assets := (sourceDirectory in Compile).value / "assets"
+unmanagedResourceDirectories in Assets ++= (if (scala.sys.env.get("SERVE_ASSETS").exists(_ == "1")) Seq(baseDirectory.value / "public") else Nil)
```

Comment:
Should be configurable to avoid packing assets for prod or development with nginx.